packages/core/scripts/mainnet/ProposeAdmin.js

```diff
@@ -0,0 +1,116 @@
+// This script proposes a contrived admin proposal to the DVM. Specifically, its a proposal to whitelist
```

Comment:
+1 for the new `mainnet` directory. Are there other scripts we want to move here?

Reply:
Probably yeah but let's move them lazily